syntax = "proto3";
package models.statsheet;

message StatSheetPossession{
    string extras = 1; 
    string offense = 2; 
    string end_time = 3;
    string result = 4; 
    string primary = 5;
    string secondary = 6; 
}

